src/site/content/en/blog/webtransport/index.md

```diff
@@ -20,104 +20,89 @@ feedback:
+The streams APIs, in contrast, provide [reliable](https://en.wikipedia.org/wiki/Reliability_(computer_networking)), ordered data transfer. They're [well-suited](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-streams) to scenarios where you need to send or receive one or more streams of ordered data. Using multiple WebTransport streams is analogous to establishing multiple [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) connections, but since HTTP/3 uses the lighter-weight [QUIC](https://www.chromium.org/quic) protocol under the hood, the can be opened and closed without as much overhead.
 
 ### Use cases
 
-This a small list of possible ways developers might use QuicTransport.
+This a small list of possible ways developers might use WebTransport.
 
 - Sending game state at a regular interval with minimal latency to a server via small, unreliable, out-of-order messages.
 - Receiving media streams pushed from a server with minimal latency, independent of other data streams.
 - Receiving notifications pushed from a server while a web page is open.
 
-As part of the origin trial process, we're interested in [hearing more](#feedback) about how you plan on using QuicTransport!
+As part of the origin trial process, we're interested in [hearing more](#feedback) about how you plan on using WebTransport!
+
+{% Aside %}
+Many of the concepts in this proposal were previously experimented with as part of the earlier [QuicTransport](https://chromestatus.com/feature/4854144902889472) Origin Trial, which did not end up being released as part of Chrome.
+
+WebTransport helps with similar use cases as QuicTransport, with the primary difference being that [HTTP/3](https://quicwg.org/base-drafts/draft-ietf-quic-http.html) is used as the underlying transport protocol, instead of using [QUIC](https://www.chromium.org/quic) directly.
+{% endAside %}
 
 ## Current status {: #status }
 
 <div class="w-table-wrapper">
 
 | Step                                       | Status                       |
 | ------------------------------------------ | ---------------------------- |
-| 1. Create explainer                        | [Complete](https://github.com/WICG/web-transport/blob/master/explainer.md) |
-| 2. Create initial draft of specification   | [Complete](https://wicg.github.io/web-transport/#quic-transport) |
+| 1. Create explainer                        | [Complete](https://github.com/w3c/webtransport/blob/main/explainer.md) |
+| 2. Create initial draft of specification   | [Complete](https://w3c.github.io/webtransport/) |
 | **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
 | **4. Origin trial**                        | [**In Progress**](#register-for-ot) |
```

Comment:
There's no origin trial for this currently is there?

Reply:
@yutakahirano just sent out the [Intent to Experiment](https://groups.google.com/a/chromium.org/g/blink-dev/c/aaLFxzw5zL4), so I assume the OT will start in the near future, in time for M91.

This PR can't be merged until the OT details are available, and ideally until @vasilvv is able to update the sample code to reflect the new interfaces.

Reply:
If this ends up in 91 beta and the samples don't get updated, can we call that out and merge anyway? I'll need to link to this article from the beta post.

Reply:
As long as the OT is active, yes, I think we should merge, and get the samples updated out-of-band.

(The updates in this won't be of much use if the OT isn't active, though, so I am still waiting to confirm that and swap in the new OT signup link.)

Reply:
I wouldn't put this in the post if the OT weren't active.

Reply:
It's now active: https://developer.chrome.com/origintrials/#/view_trial/793759434324049921